R/rdeephaven/man/AggBy.Rd

```diff
@@ -0,0 +1,101 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/agg_ops_wrapper.R
+\docType{class}
+\name{AggBy}
+\alias{AggBy}
+\title{Aggregations in Deephaven}
+\description{
+Table aggregations are a quintessential feature of Deephaven. You can apply as many aggregations as
+needed to static tables \emph{or} streaming tables, and if the parent tables are streaming, the resulting aggregated
+tables will update alongside their parent tables. It is also very easy to perform \emph{grouped} aggregations, which
+allow you to aggregate tables on a per-group basis.
+}
+\section{Apply aggregations to a table}{
+
+There are two methods for performing aggregations on a table, \code{agg_by()} and \code{agg_all_by()}. \code{agg_by()} allows you to
+perform many aggregations on specified columns, while \code{agg_all_by()} allows you to perform a single aggregation to
+every non-grouping column in the table. Both methods have an optional \code{by} parameter that is used to specify grouping columns.
+Here are some details on each method:
+\itemize{
+\item \code{TableHandle$agg_by(aggs, by)}: Creates a new table containing grouping columns and grouped data.
+The resulting grouped data is defined by the aggregation(s) specified.
+\item \code{TableHandle$agg_all_by(agg, by)}: Creates a new table containing grouping columns and grouped data.
+The resulting grouped data is defined by the aggregation specified. This method applies the aggregation to all
+non-grouping columns of the table, so it can only accept one aggregation at a time.
+}
+
+The \code{agg_by()} and \code{agg_all_by()} methods themselves do not know anything about the columns you want to perform
```

Comment:
```suggestion
The \code{agg_by()} and \code{agg_all_by()} methods themselves do not know anything about the columns on which you want to perform
```

Reply:
I think my suggested change in the previous file will actually change it here, too?